CS342MemberViews/Sources/SherryXie/SherryXie.swift

```diff
@@ -0,0 +1,70 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+import InstructorViews
+import SwiftUI
+
+
+public struct SherryXie: View {
+    @State private var pageIndex = 0
+    @Binding var navigationPath: NavigationPath
+    
+    private let pages: [Page] = Page.samplePages
+    private let dotAppearance = UIPageControl.appearance()
+    
+    public var body: some View {
+        TabView(selection: $pageIndex) {
+            ForEach(pages) { page in
+                VStack {
+                    Spacer()
+                    SherryPageView(page: page)
+                    Spacer()
+                    if page == pages.last {
+                        Button("Go back to my home page", action: goToZero)
+                            .buttonStyle(.bordered)
+                    } else if page.tag == 0 {
+                        Button("Check out the places I love!", action: incrementPage)
+                            .buttonStyle(.bordered)
+                    } else {
+                        Button("Next", action: incrementPage)
+                            .buttonStyle(.bordered)
+                    }
+                    Spacer()
```

Comment:
Great job with your view. I have noted that the buttons and the page view indicators overlap in the application. 
You can address this using a `padding` to your button. You an also remove the duplicated modifiers by packaging all the Buttons in a `Group`:
```suggestion
                    Group {
                        if page == pages.last {
                            Button("Go back to my home page", action: goToZero)
                        } else if page.tag == 0 {
                            Button("Check out the places I love!", action: incrementPage)
                        } else {
                            Button("Next", action: incrementPage)
                        }
                    }
                        .buttonStyle(.bordered)
                        .padding(.bottom, 50)
```